.github/actions/action.yml

```
@@ -0,0 +1,31 @@
+name: 'Run Tests'
+description: 'Run Tests'
+inputs:
+  test_selection:
+    description: 'Positional Argument for stestr run'
+    default: ""
+    required: false
+runs:
+  using: "composite"
+  steps:
+    - name: Clone mtreinish/qiskit-neko PUBLIC repository
+      uses: mtreinish/clone-github-repo-action@v2
+      with:
+        owner: 'mtreinish'
+        repository: 'qiskit-neko'
```

Is this the correct name? I don't think I have an actions repo `clone-github-repo-action` on my github account. Should it be `GuillaumeFalourd/clone-github-repo-action` instead? That being said I think we can just use `actions/checkout` for this too with something like:

```suggestion
    - name: Clone mtreinish/qiskit-neko PUBLIC repository
      uses: actions/checkout@v3
      with:
        repository: mtreinish/qiskit-neko
        path: qiskit-neko
```

---

Ya. I ran it again and noticed that I running the edited action in my demo repo. Sorry for the confusion and the incorrect update. Having a checkout again would be great to maintain uniformity. Have made the required changes.